tools/flowc/backends/nim/natives/string/fromCharCode.nim

```diff
@@ -5,6 +5,9 @@ proc $F_0(fromCharCode)*(code: int32): RtString =
     if (code < 0x0D800 or (0x0DFFF < code and code < 0xFFFF)):
```

Comment:
I think you should eliminate the "if (code < 0x0D800 or (0x0DFFF < code and code < 0xFFFF)):" part.
The "If code <= 0xFFFF:" should be enough.

Reply:
Fixed